src/Navigation/Home/index.css

```diff
@@ -0,0 +1,190 @@
+.home {
+  padding: 120px 15px 0px 15px;
+  color: #999999;
+  font-family: sans-serif;
+}
+.top-1 {
+  display: flex;
+  flex-direction: row;
+}
+@media only screen and (max-width: 866px) {
+  .top-1 {
+    flex-direction: column;
+    justify-content: center;
+    padding-left: 30px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.top-2 {
+    flex-direction: column;
+    justify-content: center;
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.inside-1 {
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.inside-2 {
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.inside-4 {
+    padding-top: 50px;
+    padding-right: 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.canvas {
+    padding-top: 50px;
+  }
+}
+@media only screen and (max-width: 550px) {
+  hr {
+    width: 65%;
+    margin-left: 0px;
+  }
+}
+
+@media only screen and (max-width: 550px) {
+  div.canvas {
+    width: 400px;
+  }
+}
+@media only screen and (max-width: 550px) {
+  div.inside-4 {
+    width: 400px;
+  }
+}
+@media only screen and (max-width: 490px) {
+  .home {
+    padding-left: 0px;
+  }
+}
+@media only screen and (max-width: 450px) {
+  div.top-1 {
+    padding-left: 0px;
+    margin-left: 0px;
+  }
+}
+@media only screen and (max-width: 450px) {
+  div.inside-3 {
+    padding-left: 0px;
+    margin-left: 0px;
+  }
+}
+@media only screen and (max-width: 450px) {
+  div.inside-4 {
+    padding-left: 0px;
+    margin-left: 0px;
+  }
+}
+@media only screen and (max-width: 450px) {
+  button.bob-inside {
+    padding-left: 0px;
+    margin-left: 0px;
+    width: 385px;
+  }
+}
+@media only screen and (max-width: 410px) {
+  div.canvas {
+    width: 350px;
+  }
+}
+@media only screen and (max-width: 410px) {
+  div.inside-3 {
+    width: 340px;
+  }
+}
+@media only screen and (max-width: 410px) {
+  button.bob-inside {
+    width: 350px;
+  }
+}
+@media only screen and (max-width: 410px) {
```

Comment:
perhaps we should abstract out the screen sizes in a separate constant file, so that we do not need to repeat them